app/views/enumerations/index.html.erb

```diff
@@ -103,7 +103,9 @@ See doc/COPYRIGHT.rdoc for more details.
     </div>
   <% end %>
   <div class="generic-table--action-buttons">
-    <%= link_to l(:label_enumeration_new), { action: 'new', type: klass.name , class: 'button -alt-highlight'} do %>
+    <%= link_to({ action: 'new', type: klass.name },
```

Comment:
why not use the named paths like you did in the spec: `new_enumeration_path(type: klass.name)`